war/src/main/js/section-to-tabs.js

```diff
@@ -0,0 +1,48 @@
+// Converts a page's section headings into clickable tabs, see 'About Jenkins' page for example
+const tabPanes = document.querySelectorAll(".jenkins-tab-pane")
```

Comment:
is `const` safe to use? supported by all browsers? or transpiled?

Reply:
I think yes. I use it in all JS files for a couple of years now.